packages/wrangler/src/__tests__/queues.test.ts

```diff
@@ -12,6 +14,14 @@ describe("wrangler", () => {
 	runInTempDir();
 	const std = mockConsoleMethods();
 
+	const { setIsTTY } = useMockIsTTY();
```

Comment:
Are these changes related to getting the error handling to work properly in your tests? Mind adding a comment to explain what these are for?

Reply:
Was due to trial an error. Had no effect. Will remove it.